build.sbt

```diff
@@ -25,20 +31,54 @@ def jsonDependencies(scalaVersion: String) = Seq(
   logback % Test
 ) ++ jacksons ++ specsBuild.map(_ % Test)
 
+// Common settings 
+import com.typesafe.sbt.SbtScalariform._
+import scalariform.formatter.preferences._
+
+val previousVersion = Some("2.6.0-M1") // first from this separate repo
```

Comment:
As discussed, this should be set to `None` since we make no binary compatibility guarantees between milestone versions. Once we release 2.6.0, we'll set to `Some("2.6.0")`. Then we can also remove the filters below.